src/app/reducers/user.reducer.ts

```diff
@@ -9,6 +10,8 @@ export interface State {
   user_profile: UserProfile;
   auth: any;
   selected_user_profile: UserProfile;
+  followers: Array<UserProfile>;
+  following: Array<UserProfile>;
```

Comment:
@zeus999  I don't think we discussed it this way? Since we are going to change `user_profile` to `user`. Ideally there should be `followers` and `following` in both future `user` & 'selected_user' states ? I mean how would you store the states of followers and following for both the logged in user & selected user(other user) at the same time ? @pkrawat1 @kartikjagdale  Thoughts ?